src/Controller/DashboardController.java

```diff
@@ -265,6 +267,16 @@ private void shcChangeWindows() {
             printToConsole("Successfully changed windows.");
         }
     }
+    
+    /**
+     * Toggles the Auto Mode
+     */
+    @FXML
+    private void shcLightAuto() {
+    	if (!sim.getRunning())
```

Comment:
we don't need these if (!sim.getRunning()) checks anymore since now we're using JavaFX disable.  :)

Reply:
Added a commit that resolved this!